content/en/blog/2024/otel-generative-ai/index.md

```diff
@@ -48,7 +48,7 @@ The [Semantic Conventions for Generative AI](/docs/specs/semconv/gen-ai/) focus
 on capturing insights into AI model behavior through three primary signals:
 [Traces](/docs/concepts/signals/traces/),
 [Metrics](/docs/concepts/signals/metrics/), and
-[Events](/docs/specs/otel/logs/event-api/).
+[Events](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.40.0/specification/logs/event-api.md).
```

Comment:
@drewby @lmolkova et al. - FYI, I've chosen to pin this link to the version originally being referenced in the spec, here and further below in the spec.

@open-telemetry/docs-approvers - I think that this should be a general strategy that we adopt when it comes to fixing broken links to spec pages that occur in blog posts. WDYT?

Reply:
Related:

- #4499